openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts

```diff
@@ -548,4 +548,61 @@ export const WHATS_NEW = [
       'Bug Fixes': `- Around 136 Features/Bug fixes/improvements/Tests made it into 0.12.1 release.`,
     },
   },
+  {
+    id: 10,
+    version: 'v0.13.0',
+    description: 'Released on 22 November 2022.',
+    features: [],
+    changeLogs: {
+      'Data Insights and KPIs': `-   Provides a single pane view of all the key data metrics.
+-   Analytics are provided based on the metadata metrics, the entities created, the types of entities, and the data evolution over a period of time.
+-   Admins can define the **Key Performance Indicators** (KPIs) and set goals.
+-   Goals can be set towards better documentation, ownership, and tiering.
+-   Goals are based on entities and driven to achieve targets within a specified time.
+-   **Data insights dashboard** provides a quick glance at aspects like data ownership, description coverage, data tiering, and so on.
+-   A timeseries report is provided to track progress and monitor the health of your data.
+-   Admins can view the **aggregated user activity** and get insights into user engagement and user growth.
+-   Admins can check for Daily active users and know how OpenMetadata is being used.
+-   The **Data Insights Report** is emailed weekly to assess team performance.`,
+      Glossary: `-   Introduced the functionality to **bulk upload glossary terms**.
+-   Upload a CSV with thousands of terms in one go.
+-   Define the **Owners and Reviewers** during bulk upload.
+-   Owners and Reviewers will be propagated to every glossary term.
+-   Users can **Preview** before uploading to ensure that everything looks right.
+-   Support a **Review Workflow** for new term suggestions.
+-   Suggested terms will be saved as Draft terms.
+-   Admins can review and approve these draft terms.
+-   Once approved, these terms can be used for tagging and searching.
+-   A glossary term can be moved to a different hierarchy within a glossary or across a glossary.
+-   While moving a term, even the child terms will be moved along with the parent term.`,
```

Comment:
I feel glossary is yet to done and not part of 0.13.0
cc: @harshach

Reply:
yes lets remove that